bin/kubernetes/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: metabase
+  namespace: metabase
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: "gce"
+spec:
+  tls:
+  - hosts:
+    - metabase.vikramtiwari.com
```

Comment:
Does this need to be used as a template (where each deployment fills in its host name) or can this be generalized?